snarkos/ledger/mod.rs

```diff
@@ -207,20 +213,17 @@ impl<N: Network> Ledger<N> {
 // Internal operations.
 impl<N: Network> Ledger<N> {
     /// Syncs the ledger with the network.
-    async fn initial_sync_with_network(self: &Arc<Self>) -> Result<()> {
+    pub(crate) async fn initial_sync_with_network(self: &Arc<Self>, leader_ip: &str) -> Result<()> {
```

Comment:
```suggestion
    pub(crate) async fn initial_sync_with_network(self: &Arc<Self>, leader_ip: IpAddr) -> Result<()> {
```
the `Display` impl for `IpAddr` makes it possible to use it directly in `format!`